src/find/main.rs

```diff
@@ -0,0 +1,9 @@
+extern crate glob;
```

Comment:
Copyright headers? Eurgh :-( Is there a standard template I should be copying from?

Rather more embarrassing copyright line in LICENSE now corrected.